nilearn/datasets/atlas.py

```diff
@@ -377,18 +487,32 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
         names[int(label.get('index')) + 1] = label.text
     names = list(names.values())
 
-    if not symmetric_split:
-        return Bunch(maps=atlas_img, labels=names)
-
-    if atlas_name in ("cort-prob-1mm", "cort-prob-2mm",
-                      "sub-prob-1mm", "sub-prob-2mm"):
-        raise ValueError("Region splitting not supported for probabilistic "
-                         "atlases")
-
     atlas_img = check_niimg(atlas_img)
+
     if lateralized:
         return Bunch(maps=atlas_img, labels=names)
 
+    if not symmetric_split:
+        atlas = get_data(atlas_img)
+        labels = np.unique(atlas)
+        new_label = 0
+        new_atlas = atlas.copy()
+        # Assumes that the background label is zero.
+        new_names = [names[0]]
+        for label, name in zip(labels[1:], names[1:]):
+            if name.endswith('R'):
```

Comment:
This loop works only if labels are always organized with the left version just before the right version. If this is reversed, or if they aren't consecutive this won't work. 
For Juelich, the labels are *almost* always organized like this, but there are a few exceptions. For example "WM Acoustic radiation" or "WM Inferior occipito-frontal fascicle".

Reply:
Hello @NicolasGensollen , thanks for the review! In fact, I thought about this yesterday and now It's changed in the new commit along with a small preprocessing for Juelich when is the case of symmetric_split to have the same extenstion as (left part or right part). I still have to check the last fetcher idea you mentioned as it is the logic that existed before my modifications